From: Boyuan Yang <byang@debian.org>
Date: Wed, 22 Apr 2020 10:29:39 -0400
Subject: configure.ac: Fix leftover on shell compatibility

Uses "=" instead of "==" in string comparison for POSIX
compatibility.  This is a leftover of previous similar fix.

Forwarded: https://github.com/mchehab/zbar/pull/87
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 105da09..df0220a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -845,7 +845,7 @@ dnl summary log
 echo ""
 echo "please verify that the detected configuration matches your expectations:"
 echo "------------------------------------------------------------------------"
-AS_IF([test "x$USE_NLS" == "xyes"],
+AS_IF([test "x$USE_NLS" = "xyes"],
   [echo    "gettext                $USE_NLS"]
 )
 AS_IF([test "x$win32" != "xno"],
